doc/content/devices/models/browan-tbms100/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Browan TBMS100"
+description: ""
+weight: 
+---
+
+{{< figure src="tbms100.png" alt="Generic Node Sensor Edition" class="float plain" >}}
+
+The Browan [TBMS100](https://www.browan.com/product/motion-sensor-pir/detail) is a PIR motion sensor designed to detect occupancy or movement in homes or buildings for security and efficient facility management. It helps optimize the utilization of spaces like desks, conference rooms, and bathrooms.
+
+<!--more-->
+
+# Onboarding to {{% tts %}}
```

Comment:
> We need it to be this size so it shows on the right hand menu (you can spot the difference if you deploy docs locally). Pls adjust other titles to this change

The right-hand side menu can only be visible if the page is zoomed out to 90% (for my computer). This applies to the entire site.

Reply:
hmm not sure why though, you should maybe report an issue for that? Because I can see it normally on 100%
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/da1ba16e-bdd4-488d-b264-2b159d3a9e76)